applications/luci-app-filemanager/htdocs/luci-static/resources/view/system/filemanager.js

```diff
@@ -0,0 +1,1571 @@
+            commands.push(['chown', [ownerGroup, filePath]]);
+        }
+
+        var promise = Promise.resolve();
+        commands.forEach(function(cmd) {
+            promise = promise.then(function() {
+                return fs.exec(cmd[0], cmd[1]).then(function(res) {
+                    if (res.code !== 0) {
+                        return Promise.reject(new Error(res.stderr.trim()));
+                    }
+                });
+            });
+        });
+
+        promise.then(function() {
+            ui.addNotification(null, E('p', _('Changes saved successfully.')), 'info');
+            self.loadFileList(currentPath).then(function() {
+                self.initResizableColumns();
+            });
+
+            var statusInfo = document.getElementById('status-info');
+            var statusProgress = document.getElementById('status-progress');
+            if (statusInfo) statusInfo.textContent = _('No file selected.');
+            if (statusProgress) statusProgress.innerHTML = '';
+        }).catch(function(err) {
+            ui.addNotification(null, E('p', _('Failed to save changes: %s').format(err.message)), 'error');
+        });
+    },
+handleSaveSettings: function(ev) {
+    ev.preventDefault();
+    var self = this;
+
+    var columnWidthsInput = document.getElementById('column-widths-input');
+    var columnMinWidthsInput = document.getElementById('column-min-widths-input');
+    var columnMaxWidthsInput = document.getElementById('column-max-widths-input');
+    var paddingInput = document.getElementById('padding-input');
+    var paddingMinInput = document.getElementById('padding-min-input');
+    var paddingMaxInput = document.getElementById('padding-max-input');
+    var currentDirectoryInput = document.getElementById('current-directory-input'); // Get the current directory input
+    var windowWidthInput = document.getElementById('window-width-input');
+    var windowHeightInput = document.getElementById('window-height-input');
+
+    if (columnWidthsInput && paddingInput) {
+        var columnWidthsValue = columnWidthsInput.value.trim();
+        var columnMinWidthsValue = columnMinWidthsInput.value.trim();
+        var columnMaxWidthsValue = columnMaxWidthsInput.value.trim();
+        var paddingValue = parseInt(paddingInput.value.trim(), 10);
+        var paddingMinValue = parseInt(paddingMinInput.value.trim(), 10);
+        var paddingMaxValue = parseInt(paddingMaxInput.value.trim(), 10);
+
+        if (columnWidthsValue) {
+            var widths = columnWidthsValue.split(',');
+            widths.forEach(function(widthStr) {
+                var widthParts = widthStr.split(':');
+                if (widthParts.length === 2) {
+                    var field = widthParts[0];
+                    var width = parseInt(widthParts[1], 10);
+                    if (!isNaN(width)) {
+                        config.columnWidths[field] = width;
+                    }
+                }
+            });
+        }
```

Comment:
Can surely improve this by doing something like
```js
	var inputs = {
		columnWidths: document.getElementById('column-widths-input'),
		columnMinWidths: document.getElementById('column-min-widths-input'),
		columnMaxWidths: document.getElementById('column-max-widths-input'),
		padding: document.getElementById('padding-input'),
		paddingMin: document.getElementById('padding-min-input'),
		paddingMax: document.getElementById('padding-max-input'),
		currentDirectory: document.getElementById('current-directory-input'),
		windowWidth: document.getElementById('window-width-input'),
		windowHeight: document.getElementById('window-height-input')
	};

//reusable
	function parseWidthSettings(inputValue, configKey) {
		if (!inputValue) return;
		inputValue.split(',').forEach(function(widthStr) {
			var [field, width] = widthStr.split(':');
			width = parseInt(width, 10);
			if (!isNaN(width)) {
				config[configKey][field] = width;
			}
		});
	}
...
```

And call this function for respective values